portality/static/js/edges/admin.journalarticle.edge.js

```diff
@@ -52,7 +52,7 @@ $.extend(true, doaj, {
                 // if it's not a suggestion or a journal .. (it's an article!)
                 // we really need to expose _type ...
                 var result = '<a class="delete_article_link" href="';
-                result += "/admin/article/";
+                result += "/admin/delete/article/";
```

Comment:
I realise this was like this before, but while we're at it can we treat this the same was as we do for `editArticle` - that is, take it from `doaj.adminJournalArticleSearchConfig.articleDeleteUrl` which is in turn provided in the template for this page?